tests/test_bundle_verify_meta.py

```diff
@@ -12,94 +12,49 @@
-                    "num_channels": 1,
-                    "spatial_shape": [160, 160, 160],
-                    "dtype": "float32",
-                    "value_range": [0, 1],
-                    "is_patch_data": False,
-                    "channel_def": {"0": "image"},
-                }
-            },
-            "outputs": {
-                "pred": {
-                    "type": "image",
-                    "format": "segmentation",
-                    "num_channels": 2,
-                    "spatial_shape": [160, 160, 160],
-                    "dtype": "float32",
-                    "value_range": [0, 1],
-                    "is_patch_data": False,
-                    "channel_def": {"0": "background", "1": "spleen"},
-                }
-            },
-        },
-    }
-]
+TEST_CASE_1 = [os.path.join(os.path.dirname(__file__), "testing_data", "metadata.json")]
+
+url = "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/" "meta_schema_202202281232.json"
 
 
 class TestVerifyMeta(unittest.TestCase):
     @parameterized.expand([TEST_CASE_1])
-    def test_verify(self, meta_data):
+    def test_verify(self, metafile):
         logging.basicConfig(stream=sys.stdout, level=logging.INFO)
         with tempfile.TemporaryDirectory() as tempdir:
             filepath = os.path.join(tempdir, "schema.json")
-            metafile = os.path.join(tempdir, "metadata.json")
-            with open(metafile, "w") as f:
-                json.dump(meta_data, f)
-
-            url = (
-                "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/"
-                "meta_schema_202202281232.json"
-            )
             resultfile = os.path.join(tempdir, "results.txt")
             hash_val = "486c581cca90293d1a7f41a57991ff35"
 
-            ret = os.system(
-                f"python -m monai.bundle.verify_meta -m {metafile} -u {url} -f {filepath}"
+            cmd = (
+                f"{sys.executable} -m monai.bundle.verify_meta -m {metafile} -u {url} -f {filepath}"
                 f" -r {resultfile} -v {hash_val}"
             )
+            ret = subprocess.check_call(cmd.split(" "))
             self.assertEqual(ret, 0)
             self.assertFalse(os.path.exists(resultfile))
 
+    def test_verify_error(self):
+        logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+        with tempfile.TemporaryDirectory() as tempdir:
+            filepath = os.path.join(tempdir, "schema.json")
+            metafile = os.path.join(tempdir, "metadata.json")
+            with open(metafile, "w") as f:
+                json.dump({"wrong_meta": "wrong content"}, f)
+            resultfile = os.path.join(tempdir, "results.txt")
+
+            cmd = f"{sys.executable} -m monai.bundle.verify_meta -m {metafile} -u {url} -f {filepath} -r {resultfile}"
+            with self.assertRaises(subprocess.CalledProcessError):
+                subprocess.check_call(cmd.split(" "))
```

Comment:
I had mentioned in another PR that `cmd` should be constructed as a list of strings rather than as a single string you then break apart with `split` because spaces may be in paths. This is likely in Windows because of things like "Program Files", and the error that is raised will be CalledProcessError so this issue will get masked in this test because you're trying to test an error condition using the same exception. I would suggest:

```python
cmd = [sys.executable, "-m", "monai.bundle.verify_meta", "-m", metafile, "-u", url, "-f", filepath, "-r" resultfile]
with self.assertRaises(subprocess.CalledProcessError):
    subprocess.check_call(cmd)
```

Reply:
Sure, updated in the latest commit.

Thanks for your suggestion.